diff -uprN libressl-3.2.5.orig/apps/openssl/Makefile.in libressl-3.2.5/apps/openssl/Makefile.in
--- libressl-3.2.5.orig/apps/openssl/Makefile.in	2021-03-16 05:38:14.000000000 +0900
+++ libressl-3.2.5/apps/openssl/Makefile.in	2021-03-27 19:44:00.000000000 +0900
@@ -993,31 +993,31 @@ uninstall-man: uninstall-man1
 
 
 install-exec-hook:
-	@if [ "@OPENSSLDIR@x" != "x" ]; then \
-		OPENSSLDIR="$(DESTDIR)@OPENSSLDIR@"; \
-	else \
-		OPENSSLDIR="$(DESTDIR)$(sysconfdir)/ssl"; \
-	fi; \
-	mkdir -p "$$OPENSSLDIR/certs"; \
-	for i in cert.pem openssl.cnf x509v3.cnf; do \
-		if [ ! -f "$$OPENSSLDIR/$i" ]; then \
-			$(INSTALL) -m 644 "$(srcdir)/$$i" "$$OPENSSLDIR/$$i"; \
-		else \
-			echo " $$OPENSSLDIR/$$i already exists, install will not overwrite"; \
-		fi \
-	done
+	# @if [ "@OPENSSLDIR@x" != "x" ]; then \
+	# 	OPENSSLDIR="$(DESTDIR)@OPENSSLDIR@"; \
+	# else \
+	# 	OPENSSLDIR="$(DESTDIR)$(sysconfdir)/ssl"; \
+	# fi; \
+	# mkdir -p "$$OPENSSLDIR/certs"; \
+	# for i in cert.pem openssl.cnf x509v3.cnf; do \
+	# 	if [ ! -f "$$OPENSSLDIR/$i" ]; then \
+	# 		$(INSTALL) -m 644 "$(srcdir)/$$i" "$$OPENSSLDIR/$$i"; \
+	# 	else \
+	# 		echo " $$OPENSSLDIR/$$i already exists, install will not overwrite"; \
+	# 	fi \
+	# done
 
 uninstall-local:
-	@if [ "@OPENSSLDIR@x" != "x" ]; then \
-		OPENSSLDIR="$(DESTDIR)@OPENSSLDIR@"; \
-	else \
-		OPENSSLDIR="$(DESTDIR)$(sysconfdir)/ssl"; \
-	fi; \
-	for i in cert.pem openssl.cnf x509v3.cnf; do \
-		if cmp -s "$$OPENSSLDIR/$$i" "$(srcdir)/$$i"; then \
-			rm -f "$$OPENSSLDIR/$$i"; \
-		fi \
-	done
+	# @if [ "@OPENSSLDIR@x" != "x" ]; then \
+	# 	OPENSSLDIR="$(DESTDIR)@OPENSSLDIR@"; \
+	# else \
+	# 	OPENSSLDIR="$(DESTDIR)$(sysconfdir)/ssl"; \
+	# fi; \
+	# for i in cert.pem openssl.cnf x509v3.cnf; do \
+	# 	if cmp -s "$$OPENSSLDIR/$$i" "$(srcdir)/$$i"; then \
+	# 		rm -f "$$OPENSSLDIR/$$i"; \
+	# 	fi \
+	# done
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -uprN libressl-3.2.5.orig/configure libressl-3.2.5/configure
--- libressl-3.2.5.orig/configure	2021-03-16 05:38:12.000000000 +0900
+++ libressl-3.2.5/configure	2021-03-27 19:52:28.000000000 +0900
@@ -13115,16 +13115,64 @@ else
 fi
 
  if test "x$ac_cv_func_strndup" = xyes; then
-  HAVE_STRNDUP_TRUE=
-  HAVE_STRNDUP_FALSE='#'
+  if test x$HOST_OS = xdarwin; then
+    # Check availability
+    fn_name="strndup"
+    fn_min_version="10.7"
+    { $as_echo "$as_me:${as_lineno-$LINENO}: checking availability of ${fn_name}" >&5
+      $as_echo_n "checking availability of ${fn_name}... " >&6; }
+    if test -n ${MACOSX_DEPLOYMENT_TARGET} \
+      && ! printf '%s\n%s' "${fn_min_version}" "${MACOSX_DEPLOYMENT_TARGET}" | sort -C -V; then
+      {
+        $as_echo "$as_me:${as_lineno-$LINENO}: unavailable" >&5
+        $as_echo "only available on macOS ${fn_min_version} or newer" >&6
+      }
+      HAVE_STRNDUP_TRUE='#'
+      HAVE_STRNDUP_FALSE=
+    else
+      {
+        $as_echo "$as_me:${as_lineno-$LINENO}: available" >&5
+        $as_echo "available" >&6
+      }
+      HAVE_STRNDUP_TRUE=
+      HAVE_STRNDUP_FALSE='#'
+    fi
+  else
+    HAVE_STRNDUP_TRUE=
+    HAVE_STRNDUP_FALSE='#'
+  fi
 else
   HAVE_STRNDUP_TRUE='#'
   HAVE_STRNDUP_FALSE=
 fi
 
  if test "x$ac_cv_func_strnlen" = xyes; then
-  HAVE_STRNLEN_TRUE=
-  HAVE_STRNLEN_FALSE='#'
+  if test x$HOST_OS = xdarwin; then
+    # Check availability
+    fn_name="strnlen"
+    fn_min_version="10.7"
+    { $as_echo "$as_me:${as_lineno-$LINENO}: checking availability of ${fn_name}" >&5
+      $as_echo_n "checking availability of ${fn_name}... " >&6; }
+    if test -n ${MACOSX_DEPLOYMENT_TARGET} \
+      && ! printf '%s\n%s' "${fn_min_version}" "${MACOSX_DEPLOYMENT_TARGET}" | sort -C -V; then
+      {
+        $as_echo "$as_me:${as_lineno-$LINENO}: unavailable" >&5
+        $as_echo "only available on macOS ${fn_min_version} or newer" >&6
+      }
+      HAVE_STRNLEN_TRUE='#'
+      HAVE_STRNLEN_FALSE=
+    else
+      {
+        $as_echo "$as_me:${as_lineno-$LINENO}: available" >&5
+        $as_echo "available" >&6
+      }
+      HAVE_STRNLEN_TRUE=
+      HAVE_STRNLEN_FALSE='#'
+    fi
+  else
+    HAVE_STRNLEN_TRUE=
+    HAVE_STRNLEN_FALSE='#'
+  fi
 else
   HAVE_STRNLEN_TRUE='#'
   HAVE_STRNLEN_FALSE=
@@ -13380,8 +13428,32 @@ else
 fi
 
  if test "x$ac_cv_func_timingsafe_bcmp" = xyes; then
-  HAVE_TIMINGSAFE_BCMP_TRUE=
-  HAVE_TIMINGSAFE_BCMP_FALSE='#'
+  if test x$HOST_OS = xdarwin; then
+    # Check availability
+    fn_name="timingsafe_bcmp"
+    fn_min_version="10.12.1"
+    { $as_echo "$as_me:${as_lineno-$LINENO}: checking availability of ${fn_name}" >&5
+      $as_echo_n "checking availability of ${fn_name}... " >&6; }
+    if test -n ${MACOSX_DEPLOYMENT_TARGET} \
+      && ! printf '%s\n%s' "${fn_min_version}" "${MACOSX_DEPLOYMENT_TARGET}" | sort -C -V; then
+      {
+        $as_echo "$as_me:${as_lineno-$LINENO}: unavailable" >&5
+        $as_echo "only available on macOS ${fn_min_version} or newer" >&6
+      }
+      HAVE_TIMINGSAFE_BCMP_TRUE='#'
+      HAVE_TIMINGSAFE_BCMP_FALSE=
+    else
+      {
+        $as_echo "$as_me:${as_lineno-$LINENO}: available" >&5
+        $as_echo "available" >&6
+      }
+      HAVE_TIMINGSAFE_BCMP_TRUE=
+      HAVE_TIMINGSAFE_BCMP_FALSE='#'
+    fi
+  else
+    HAVE_TIMINGSAFE_BCMP_TRUE=
+    HAVE_TIMINGSAFE_BCMP_FALSE='#'
+  fi
 else
   HAVE_TIMINGSAFE_BCMP_TRUE='#'
   HAVE_TIMINGSAFE_BCMP_FALSE=
